flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamGraphGeneratorTest.java

```diff
@@ -138,138 +132,6 @@ public void testBufferTimeout() {
 		}
 	}
 
-	/**
```

Comment:
This one was not just testing "split"

Reply:
Right, I missed the bit with testing partitioning pre union. I reverted and adjusted the test.